contrib/setup-win32.nsi

```diff
@@ -0,0 +1,135 @@
+#!Nsis Installer Command Script
+#
+# To build an installer from the script you would normally do:
+#
+#   dnf install mingw32-nsis
+#   makensis setup-win32.nsi
+
+Name ""
+OutFile "fwupd-setup-x86_64.exe"
```

Comment:
In the same vain as other parts of this file that I think should be dynamic - this probably should be something like `fwupd-$(git describe)-setup-x86_64.exe`.

Reply:
If it is difficult to do it dynamically in this file, a simple `mv` afterwards is fine.

Reply:
How would that work with `store_artifacts`... Can we include a blob there?

Reply:
Can certainly include a directory if globs don't work.

Reply:
/me tries...